example/lib/src/home.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+import '../utils/routes_constants.dart';
+import '../utils/strings_constants.dart';
+import '../utils/text_styles.dart';
+import '../utils/constants.dart';
+
+class Home extends StatefulWidget {
+  const Home({Key? key}) : super(key: key);
+
+  @override
+  _HomeState createState() => _HomeState();
+}
+
+class _HomeState extends State<Home> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          StringsConstants.homeTitle,
+        ),
+      ),
+      body: Center(
+        child: Column(
+          children: <Widget>[
+            Padding(
+              padding: const EdgeInsets.all(
+                Constants.padding,
+              ),
+              child: Text(
+                StringsConstants.homeSubtitle,
+                style: TextStyles.styleTitleHome,
+              ),
+            ),
+            _paddingButton(
+              StringsConstants.titleScreen,
+              RoutesConstants.titleRoute,
+            ),
+            _paddingButton(
+              StringsConstants.imageScreen,
+              RoutesConstants.imageRoute,
+            ),
+            _paddingButton(
+              StringsConstants.informationScreen,
+              RoutesConstants.informationRoute,
+            ),
+            _paddingButton(
+              StringsConstants.overviewScreen,
+              RoutesConstants.overviewRoute,
+            ),
+            _paddingButton(
+              StringsConstants.buttonScreen,
+              RoutesConstants.buttonRoute,
+            ),
```

Comment:
You can create a class that contains a String relative to text of the widget and a String relative to the route that you want to use, and then instead of repeat a X number of _paddingButton, create a ListBuilder with only one _paddingButton creator

Reply:
done